cloud/amazon/ec2_vpc_nat_gateway.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import time
+import datetime
+try:
+    import boto3
+    import botocore
+except ImportError:
+    module.fail_json(msg='boto3 and botocore are required.')
```

Comment:
You shouldn't call module.fail_json here. In fact, module isn't even defined at this point. You should set a boolean value here indicating that boto3 and botocare aren't available, and then in your main method call module.fail_json